content/md/enterprise-repository/1-get-enterprise-edition-archive.md

```diff
@@ -95,6 +95,36 @@ The authorization is effective soon after the change on the project console.
 During this time, you can see that the action is in progress with the status `Activation pending`.
 :::
 
+## Add access to your enterprise repository to a new contact
+
+Supervisors or Project Managers have the possibility to **create a new contact** from this form. It is possible to attach this contact directly to your company or to declare it as external.
+The external contact will be visible from the contacts of your company and will be removed from your company if it is no longer linked to your company. For example, if you delete its access.
```

Comment:
The external contact will be visible within the contacts of your company and will be removed from your company if they are longer linked to your company. For example, if you delete their access.